uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Form/PublicHoliday.php

```diff
@@ -8,4 +8,189 @@
  * @see http://wiki.civicrm.org/confluence/display/CRMDOC43/QuickForm+Reference
  */
 class CRM_HRLeaveAndAbsences_Form_PublicHoliday extends CRM_Core_Form {
+
+  private $defaultValues = [];
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setDefaultValues() {
+    if(empty($this->defaultValues)) {
+      if ($this->_id) {
+        $this->defaultValues = CRM_HRLeaveAndAbsences_BAO_PublicHoliday::getValuesArray($this->_id);
+      } else {
+        $this->defaultValues = [
+          'id' => null,
+          'title' => '',
+          'date' => '',
+          'is_active' => 1,
+        ];
+      }
+    }
+
+    return $this->defaultValues;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildQuickForm() {
+    $this->_id = CRM_Utils_Request::retrieve('id' , 'Positive', $this);
+
+    $this->addFields();
+    $this->addFieldsRules();
+
+    $this->addButtons($this->getAvailableButtons());
+    $this->assign('deleteUrl', $this->getDeleteUrl());
+
+    CRM_Core_Resources::singleton()->addStyleFile('uk.co.compucorp.civicrm.hrleaveandabsences', 'css/hrleaveandabsences.css');
+    CRM_Core_Resources::singleton()->addStyleFile('uk.co.compucorp.civicrm.hrleaveandabsences', 'css/spectrum.css');
+    CRM_Core_Resources::singleton()->addScriptFile('uk.co.compucorp.civicrm.hrleaveandabsences', 'js/spectrum-min.js', CRM_Core_Resources::DEFAULT_WEIGHT, 'html-header');
```

Comment:
We don't need any of these resources. This form doesn't use any of the rules in hrleaveabsences.css and spectrum is only used by the AbsenceType form